src/iOS/Xamarin.iOS/Samples/Search/FindPlace/FindPlace.cs

```diff
@@ -310,359 +290,362 @@ private void LocationDisplay_LocationChanged(object sender, Esri.ArcGISRuntime.L
+            // Search for the graphics underneath the user's tap.
             IReadOnlyList<IdentifyGraphicsOverlayResult> results = await _myMapView.IdentifyGraphicsOverlaysAsync(e.Position, 12, false);
 
-            // Clear callouts and return if there was no result
-            if (results.Count < 1 || results.First().Graphics.Count < 1) { _myMapView.DismissCallout(); return; }
+            // Clear callouts and return if there was no result.
+            if (results.Count < 1 || results.First().Graphics.Count < 1)
+            {
+                _myMapView.DismissCallout();
+                return;
+            }
 
-            // Get the first graphic from the first result
+            // Get the first graphic from the first result.
             Graphic matchingGraphic = results.First().Graphics.First();
 
-            // Get the title; manually added to the point's attributes in UpdateSearch
-            String title = matchingGraphic.Attributes["Match_Title"] as String;
+            // Get the title; manually added to the point's attributes in UpdateSearch.
+            string title = matchingGraphic.Attributes["Match_Title"] as string;
 
-            // Get the address; manually added to the point's attributes in UpdateSearch
-            String address = matchingGraphic.Attributes["Match_Address"] as String;
+            // Get the address; manually added to the point's attributes in UpdateSearch.
+            string address = matchingGraphic.Attributes["Match_Address"] as string;
 
-            // Define the callout
+            // Define the callout.
             CalloutDefinition calloutBody = new CalloutDefinition(title, address);
 
-            // Show the callout on the map at the tapped location
+            // Show the callout on the map at the tapped location.
             _myMapView.ShowCalloutAt(e.Location, calloutBody);
         }
 
         /// <summary>
-        /// Returns a list of suggestions based on the input search text and limited by the specified parameters
+        /// Returns a list of suggestions based on the input search text and limited by the specified parameters.
         /// </summary>
-        /// <param name="searchText">Text to get suggestions for</param>
-        /// <param name="location">Location around which to look for suggestions</param>
+        /// <param name="searchText">Text to get suggestions for.</param>
+        /// <param name="location">Location around which to look for suggestions.</param>
         /// <param name="poiOnly">If true, restricts suggestions to only Points of Interest (e.g. businesses, parks),
-        /// rather than all matching results</param>
-        /// <returns>List of suggestions as strings</returns>
-        private async Task<IEnumerable<String>> GetSuggestResults(string searchText, string location = "", bool poiOnly = false)
+        /// rather than all matching results.</param>
+        /// <returns>List of suggestions as strings.</returns>
```

Comment:
Since we choose to document this method, we might as well make docs accurate. Mention the fact that we may return `null` from here, e.g.

> _List of suggestions as strings; or null if suggestions could not be retrieved._